script/export_dmf_eligible_consumers.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+# This script generates a CSV report which lists all consumers eligible to be included in a DMF call
+
+# To run this script
+# bundle exec rails runner script/export_dmf_eligible_consumers.rb
+
+require "#{Rails.root}/app/domain/operations/families/verifications/dmf_determination/dmf_utils.rb"
+include ::Operations::Families::Verifications::DmfDetermination::DmfUtils # rubocop:disable Style/MixinUsage
+
+if EnrollRegistry.feature_enabled?(:alive_status)
+  csv_headers = [
+    "Family Hbx ID",
+    "Person Hbx ID",
+    "Person First Name",
+    "Person Last Name",
+    "Enrollment Hbx ID",
+    "Enrollment Type",
+    "Enrollment Status"
+  ].freeze
+
+  dmf_eligibile_members = []
+
+  Family.enrolled_members_with_ssn.no_timeout.each do |family|
+    family.family_members.each do |member|
+      eligibility_states = member_dmf_determination_eligible_enrollments(member, family)
+      next unless eligibility_states
+      next unless AcaEntities::Operations::EncryptedSsnValidator.new.call(member.person.encrypted_ssn).success?
```

Comment:
- This operation does not work for model object encrypted ssn. can you reverify.